src/middlewares/AuthCheck.ts

```diff
@@ -0,0 +1,21 @@
+import UserService from "../services/UserService";
```

Comment:
This class seems to not be used anywhere. Furthermore, it seems the logic can be moved to JWTCheckMiddleware.